src/length-value.js

```diff
@@ -50,15 +50,29 @@
   LengthValue.prototype.add = function(addedLength) {
     if (this instanceof SimpleLength && addedLength instanceof SimpleLength && this.type == addedLength.type) {
       return this._addSimpleLengths(addedLength);
-    } 
-    throw new TypeError('Not implemented yet');
+    } else if (addedLength instanceof LengthValue) {
+      //ensure both lengths are of type CalcLength before adding
+      var lengthToAdd_1 = this._asCalcLength();
+      var lengthToAdd_2 = addedLength._asCalcLength();
+
+      return lengthToAdd_1._addCalcLengths(lengthToAdd_2);
```

Comment:
I think Eddy might have mentioned this in a previous review. You can just say
```
return this._asCalcLength()._addCalcLengths(addedLength._asCalcLength());
```
Also, for future reference, please don't use underscores in any identifier names except for object members that you wish to be treated as private. E.g. here you would use lengthToAdd1, lengthToAdd2.